src/main/java/frc/robot/subsystems/scoring/HoodIOSparkFlex.java

```diff
@@ -15,8 +16,10 @@ public class HoodIOSparkFlex implements HoodIO {
 
     double goalAngleRad = 0.0;
 
+    private Timer homeTimer = new Timer();
+
     public HoodIOSparkFlex() {
-        hoodMotor.setSmartCurrentLimit(10);
+        hoodMotor.setSmartCurrentLimit(120);
```

Comment:
We can probably lower this since it is the hood

Reply:
I think we want to gain schedule this one or something. For moving between positions and holding in stow position we don't need much torque but I'm concerned about how much torque we will need to hold the amp hood in position when a note is passing through it.

Reply:
Hopefully we can test this tonight to see if it will be a problem or not